release/internal/docs/data/key.pub

```diff
@@ -0,0 +1 @@
+ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBH50uLmbHk8aommqDUZp1VJSm/KbcESVY3LpnYO+Dmmw428fM3KMuPnu7MBJ4Dfl8hc7ubirjJHK2oh6kelcHfI=
```

Comment:
We shouldn't check in any keys - public or not.

Reply:
this was the solution proposed from @Behnam-Shobiri , perharps he can chime in on this

Reply:
@caseydavenport we can put the public key in the CI secret so that the rotation would be easier.

Reply:
@radTuti was this key rotated before adding to CI? This key that was published here should be revoked/not work.

Reply:
@anthonytwh FYI this is only the public key. Although we can always rotate